test/libsolidity/semanticTests/storage/mappings_array2d_pop_delete.sol

```diff
@@ -0,0 +1,39 @@
+contract C {
+    mapping (uint => uint)[][] a;
+
+	function n1(uint key, uint value) public {
+		a.length++;
+		mapping (uint => uint)[] storage b = a[a.length - 1];
+		b.length++;
+		b[b.length - 1][key] = value;
+	}
+
+	function n2() public {
+		a.length++;
+		mapping (uint => uint)[] storage b = a[a.length - 1];
+		b.length++;
+	}
+
+	function map(uint key) public view returns (uint) {
+		mapping (uint => uint)[] storage b = a[a.length - 1];
+		return b[b.length - 1][key];
+	}
```

Comment:
Note to self: This returns the value stored in the array's last dimension's last slot at `key`